Contact.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace HwCreateGame
+{
+    public class Contact
+    {
+        public string Name { get; set; }
```

Comment:
At this point I expect you to use proper methods for your properties. Why does it have a public set is there a reason?